arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsManagementClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/alertrules/{ruleName}/incidents/{incidentName}": {
+      "get": {
+        "description": "Gets an incident associated to an alert rule",
+        "tags": [
+          "Alert"
+        ],
+        "operationId": "AlertRuleIncidents_Get",
```

Comment:
This seems to be a weird naming convention. `alertrules` is the parent resource type and `incidents` seems to be a child resource type.
So, ideally the operationId should be `Incidents_Get`